pages/team.js

```diff
@@ -112,6 +112,34 @@ export default function Team() {
                 img="/team/liv.png"
                 pronouns="she/her"
               />
+              <Bio
+                name="Rick Blount"
+                teamRole="VP of Philanthropy"
+                text="Rick Blount was previously helped raise $19.5 million for a brand-new YMCA in Burlington, Vermont. Before that, he worked at the University of Vermont’s medical school in communications and alumni/fundraising; and has been a journalist–helping start Consumer Reports’ health publication. When he was 25, he taught English for a year near Wuhan, China."
+                img="/team/rick.png"
+                pronouns="he/him"
+              />
+              <Bio
+                name="Thea Baren"
+                teamRole="Executive Assistant to the COO"
+                text="Thea will now serve as Christina’s executive assistant."
+                img="/team/thea.png"
+                pronouns="she/her"
+              />
+              <Bio
+                name="Bence"
+                teamRole="Bookkeeper"
+                text="Bence is responsible for  keeping accurate financial books for hq and every org on hack club bank (40,000+ transactions in the last year!)."
+                img="/team/bence.png"
+                pronouns="he/him"
+              />
+               <Bio
+                name="Chris Walker"
+                teamRole="Engineer"
+                text="Chris Walker is joining us part-time from San Francisco to ship sinerider and bring mayhem and magic to #gamedev and beyond."
```

Comment:
Ideally "siderider" would be linked to a page explaining what it is

Reply:
Is there a website to siderider? @cjdenio

Reply:
Chris is in the Acknowledgements section already; maybe we move that card here and add the new copy?